application/src/main/java/org/thingsboard/server/service/mail/TbMailSender.java

```diff
@@ -70,14 +68,31 @@ public TbMailSender(TbMailContextComponent ctx, JsonNode jsonConfig) {
         setJavaMailProperties(createJavaMailProperties(jsonConfig));
     }
 
-    @SneakyThrows
     @Override
-    public void doSend(MimeMessage[] mimeMessages, @Nullable Object[] originalMessages) {
+    protected void doSend(MimeMessage[] mimeMessages, @Nullable Object[] originalMessages) {
```

Comment:
Protected is not good for test mocking. Package-private is the best for testing, public is also good when it is possible